src/Stores/Politicians/reducer.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+import type { PoliticianType, Action } from '../../types';
+
+export type State = {
+  +politicians: ?Array<PoliticianType>,
+  +loading: boolean,
+  +error: *,
+};
+
+const initialState = {
+  politicians: null,
```

Comment:
Which you can see here https://github.com/veritus/veritus-web/pull/46/files#diff-9c4839bf4b44852f9b55191908e9e8b4R6

`export const politiciansLoaded = (state: State) => state.politicians.politicians !== null;`